### ISSUE TITLE
As a warehouse  I want  to manager Inventory

### ISSUE THREAD

**Issue by BorderITS** [borderitsolutions/amadaa#39]

**Comment by lcabrini** [comment]
I wish we can clarify this story a little bit. I see a few activities that could go into this:

1. as a warehouse manager I want to transfer stock to a different location
2. as a warehouse manager I want to receive a stock transfer
3. as a warehouse manager I want to request stock to be transferred to me

The wording may not be that good, but these are the different stock transfer activities that I can think of. Each should be a user story.

**Comment by BorderITS** [comment]
ok

**Comment by lcabrini** [comment]
Yes, that makes it a little more explicit.

**Comment by BorderITS** [comment]
ok

**Comment by lcabrini** [comment]
I just realized that I still don't understand something. I assume that we both mean "transfer stock to a different branch", right?

Isn't is also possible that a warehouse manager would want to move stock to a different warehouse that belongs to the same branch? If so, we would have to clarify even further.

**Comment by BorderITS** [comment]
Yes that remains me he made mention something like that so should create a user story on that

**Comment by lcabrini** [comment]
Yes, we will probably need one user story for each of the possible transfers. At least that is what I think.

**Comment by BorderITS** [comment]
OK I will create an issue right now on that
